IAMOTZ/ievents
# #157720609 Fix UI bugs

#### What does this PR do?
It fixes most of the UI bugs that exist in the application.
#### Description of Task to be completed?
Fix UI bugs.
The current user interface has bugs that would make it difficult for Users to enjoy the value that this platform has to offer. This Pull Request is meant to fix all the major bugs that are identified.
#### How should this be manually tested?
- clone the app and change into the project directory
- setup the app(instructions in the readme)
- run the tests to ensure that nothing is failing/breaking
- visit the app in a browser
- explore the various functionalities of the app, everything should still be working fine but with a better User Interface
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#157720609 
#### Questions:
None
